src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -49,6 +88,7 @@ The properties are:
 * __name__: A unique name for your adapter, used for logging.
 * __serverEntrypoint__: The entrypoint for server-side rendering.
 * __exports__: An array of named exports when used in conjunction with `createExports` (explained below).
+* __supportedAstroFeatures__: A map of Astro built-in features. Sometime adapters can't support those features, and maybe they want to deprecate them.
```

Comment:
```suggestion
* __supportedAstroFeatures__: A map of Astro built-in features. This allows Astro to determine which features an adapter is unable or unwilling to support so appropriate error messages can be provided.
```
I'm not sure if this is true, but it sounds good!  :sweat_smile: Maybe something like this?

Reply:
That's a very good suggestion! Thank you!